Clarify Facebook bridges status

somini opened the pull request:
The fact that there were two bridges and one is deprecated should be
made more clear.
This preserves all existing URL.

This also updates all URL to secure HTTPS versions.

---

I was using the legacy Facebook bridge myself, since the new one was using insecure URI.

somini committed:
Clarify Facebook bridges status

Distinguish between both Facebook bridges by their title.
This preserves all existing URL.

This also updates all URL to secure HTTPS versions.

somini committed:
Configure author name abbreviation

triatic commented:
Both bridges work as well as each other. As I understand it, FB2 was created to fix a captcha issue with the original which now seems not to be a problem anymore.

I'm currently using the original which is working fine. Keeping FB2 as a backup bridge rather than deprecating the original seems reasonable to me.

somini commented:
https://github.com/RSS-Bridge/rss-bridge/issues/1188#issuecomment-508914802

It seems the original Facebook bridge still works, but it's not being maintained.

triatic commented:
I added a PR to fix the issue reported there (remove relative times/date) which has been merged.

In my opinion it's better to maintain the bridge that pulls from the more "standard" url www.facebook.com rather than touch.facebook.com. Of course I'll leave that decision to the project maintainers.

somini commented:
I agree, let's keep both bridges. I reworded this to remove the "deprecated" references, but still distinguish between both bridges more easily.

This also updates all URL to HTTPS.

triatic commented:
Sounds good to me.

Line 295 states "v2", perhaps it should be "Touch Site" for consistency?

somini committed:
Improve feed names

Use the correct feed name on each bridge.
Make sure the feed names don't repeat the "Facebook" name.

Thanks @triatic

somini commented:
#1283 someone is confused about having two Facebook bridges.

triatic commented:
If one is to be dropped, I suggest dropping FB2. We should also note that FacebookBridge is enabled by default upon installation, so I would suggest it is still working well for the vast majority (including myself).

somini commented:
The PR just clarifies the names of the bridges and forces HTTPS urls everywhere.

There's no need to drop working bridges, we should just have better names than "Facebook" and "F2", so that bug reports are clearer.

triatic commented:
It was mooted before by @LogMANOriginal before that the two Facebook bridges should be consolidated into one somehow. I agree with the person you referenced that the situation is confusing, and does raise the prospect of people seeking Facebook Bridge support when in fact they are using FB2.

Even when renamed, we still run the possibility of people referring to any bridge that accesses Facebook as "Facebook Bridge".

logmanoriginal commented:
Very interesting discussion. A single Facebook bridge would certainly reduce confusion, but that bridge would also have to pull from multiple sources to make it work for everyone. Certainly possible, but only if someone is willing to spend time on it.

This is not something I'd be willing to spend time on right now, especially considering that I don't use Facebook (as does @teromene IIRC). Let me know if anyone of you is interested in taking on this task. Unfortunately there are no statistics, but chances are high that the Facebook bridges are the most utilized bridges in the repository.

Anyway, this PR looks good to me so I'm going to merge.

triatic commented:
Merging Facebook and FB2 not only sounds like a messy task, but the complexity of such code sounds like a recipe for headaches.

In my low-volume use case Facebook does not trip any limits. If FB and FB2 can handle equal volumes of traffic without triggering a captcha, the need for FB2 seems questionable.

P.S. Good to see you again @LogMANOriginal 👍 I noticed you'd not posted for a while.

logmanoriginal commented:
> If FB and FB2 can handle equal volumes of traffic without triggering a captcha, the need for FB2 seems questionable.

I'll leave that for @teromene to decide. Don't hesitate to open an issue for it.

> P.S. Good to see you again @LogMANOriginal +1 I noticed you'd not posted for a while.

Thanks, I'm happy to be back as well :grin:
